vendor/github.com/containers/image/types/types.go

```diff
@@ -292,7 +305,7 @@ type ManifestUpdateInformation struct {
 // for other manifest types.
 type ImageInspectInfo struct {
 	Tag           string
-	Created       time.Time
+	Created       *time.Time
```

Comment:
it seems like this change breaks the tests. 
I think we need to update `cmd/buildah/common.go` line 116 to
```
created = *inspectInfo.Created
```
and `cmd/buildah/images.go` line 198 to:
```
date := *inspect.Created
```